README.md

```diff
@@ -15,6 +15,19 @@ The Standard for Public Code is in a draft format.
 We are preparing it for a version 1.0 release.
 Currently, we are testing it on a small number of codebases.
 
+## Applying the Standard for Public Code on your codebase
+
+If you want to apply the Standard for Public Code to your codebase, just go ahead, it's an open standard and free for anyone to use.
+To see how ready your codebase is, you can do a quick [eligibility self assessment](https://publiccodenet.github.io/assessment-eligibility) that will give you some rough expectations on how much work you may need to do to meet all criteria.
+
+The standard *should* be mostly self-explanatory in how to apply it on your codebase.
+If anything in the standard is unclear, we encourage you to open an issue here so that we can help you and everyone else who feel the same as you.
```

Comment:
```suggestion
If anything in the standard is unclear, we encourage you to open an issue here so that we can help you and anyone else who feels the same as you.
```